static/js/main.js

```diff
@@ -19,8 +19,8 @@ function findCurrentSection(htmlEl, mainEl) {
     // caught in a margin, don't change the highlight
     return null;
   } else {
-    // above or below main; clamp
-    targetEl = scrollPct < 0.5 ? mainEl.firstElementChild : mainEl.lastElementChild;
+    // above or below main; this is unlikely with non-dynamic scroll percent
```

Comment:
Once we've changed the algorithm, there's no reason to mention a dynamic scroll percent. How about:

```
// above main; just highlight the first possibility
```